src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import classNames from 'classnames';
+import { useMemo } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { ColumnSortLink } from '../ColumnSortLink';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+  const sexParam = searchParams.get('sex');
+  const query = searchParams.get('query');
+  const centuries = searchParams.getAll('centuries');
+
+  const visiblePeople = useMemo(() => (
+    people.filter(person => {
+      const isSexFilter = sexParam
+        ? person.sex === sexParam
+        : true;
+      const isQueryFilter = query
+        ? person.name.toLowerCase().includes(query)
```

Comment:
It's a good practice to also convert the `query` to lowercase when comparing strings, to make the search case-insensitive. Instead of `query`, use `query.toLowerCase()`.